### PR TITLE
Updated some files

### PR DESCRIPTION
**Pull request by CubE135**
Hey, this is just a test :)

### CODE REVIEW

**Review by CubE3011 — COMMENTED** [review]
I like that!

File: assets/main.css [hunk 1]
```diff
@@ -1,3 +1,4 @@
 body{
     background: red;
+    color: blue;
```
Comment (CubE3011) [inline assets/main.css:3]: I love the color blue!

File: index.html [hunk 2]
```diff
@@ -1 +1,2 @@
-This is a test!
\ No newline at end of file
+This is a test!
+Fixed some stuff!
```
Comment (CubE3011) [inline index.html]: Could you add another line?

Comment (CubE135) [reply index.html]: Done :)

**Pull request merged**